bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/module/provider/AnnotationActionModuleTypeHelper.java

```diff
@@ -39,18 +39,21 @@
 import org.openhab.core.automation.type.Input;
 import org.openhab.core.automation.type.ModuleTypeProvider;
 import org.openhab.core.automation.type.Output;
+import org.openhab.core.automation.util.mapper.ActionInputsToConfigDescriptionParameters;
 import org.openhab.core.config.core.ConfigDescriptionParameter;
 import org.openhab.core.config.core.ConfigDescriptionParameter.Type;
 import org.openhab.core.config.core.ConfigDescriptionParameterBuilder;
 import org.openhab.core.config.core.Configuration;
 import org.openhab.core.config.core.ParameterOption;
+import org.openhab.core.i18n.UnitProvider;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * Helper methods for {@link AnnotatedActions} {@link ModuleTypeProvider}
  *
  * @author Stefan Triller - Initial contribution
+ * @author Florian Hotze - Added configuration description parameters for thing modules
```

Comment:
Wouldn't it make sense to make this a component and inject the `UnitProvider` directly? It keeps a simpler API and all consumers are components anyway, so we could inject the helper there instead of creating a new instance and injecting the `UnitProvider`to the module type providers.

Reply:
If all consumers ar really components, yes that's a great idea and I should have done that way. I will check again.

Reply:
Creating a component for `AnnotationActionModuleTypeHelper` seems doable.?

Reply:
Done